.golangci.yml

```diff
@@ -33,30 +33,31 @@ linters:
     #- gocyclo
     #- gofmt
     - goimports
-    - golint
+    # - golint
     #- gomnd
     #- goprintffuncname
     #- gosec
     #- gosimple
     - govet
     - ineffassign
-    - interfacer
+    # - interfacer
     #- lll
-    - maligned
+    # - maligned
```

Comment:
I don't think we are running the replacement linter for this one.
According to the `golangci-lint` warning: "Replaced by govet 'fieldalignment'"

If I make the following modification without the PR I get an linter error, but not after the PR.

```
diff --git a/command.go b/command.go
index 9d5e9cf..943bb17 100644
--- a/command.go
+++ b/command.go
@@ -175,9 +175,6 @@ type Command struct {
 	// helpCommandGroupID is the group id for the helpCommand
 	helpCommandGroupID string

-	// completionCommandGroupID is the group id for the completion command
-	completionCommandGroupID string
-
 	// versionTemplate is the version template defined by user.
 	versionTemplate string

@@ -194,6 +191,8 @@ type Command struct {
 	// CompletionOptions is a set of options to control the handling of shell completion
 	CompletionOptions CompletionOptions

+	// completionCommandGroupID is the group id for the completion command
+	completionCommandGroupID string
 	// commandsAreSorted defines, if command slice are sorted or not.
 	commandsAreSorted bool
 	// commandCalledAs is the name or alias value used to call this command.
```

Reply:
Yeah I think we'd need to configure govet to do that: https://golangci-lint.run/usage/linters/#govet. I tend to just use:

```yaml
  govet:
    enable-all: true
```

But if we've not run that before it might trigger a load of stuff